Extension/package.nls.json

```diff
@@ -1,4 +1,10 @@
 {
+    "c_cpp.subheaders.intelliSense.title" : "IntelliSense Features",
+    "c_cpp.subheaders.codeAnalysis.title" : "Code Analysis",
+    "c_cpp.subheaders.formatting.title" : "Formatting",
+    "c_cpp.subheaders.doxygen.title" : "Code Documentation",
+    "c_cpp.subheaders.intelliSenseDefaults.title" : "IntelliSense Default Config.",
```

Comment:
The period here seems odd.  Also, keep in mind that this will be translated, and other languages may not abbreviate this or have different abbreviation rules.  Maybe use "IntelliSense Default Configuration".

Reply:
Yeah, there shouldn't be a period, right?